rune/libenclave/internal/runtime/pal/skeleton/skeleton_remote_attestation_with_rune.md

```diff
@@ -0,0 +1,55 @@
+# Introduction
+This guide will tell you how to use remote attestation of Intelsgx in skeleton with rune.
+
+# Before you start
+- Firstly, you need to build a skeleton bundle according to [this guide](https://github.com/alibaba/inclavare-containers/blob/master/rune/libenclave/internal/runtime/pal/skeleton/README.md) from scratch.
+- Secondly, you need to build rune according to [this guide](https://github.com/alibaba/inclavare-containers#rune).
+- Finally, you also need to register a `SPID` and `Subscription Key` of [IAS](https://api.portal.trustedservices.intel.com/EPID-attestation). After the registration, Intel will respond with a SPID which is needed to communicate with IAS.
+
+# Enable Remote Attestation when skeleton starts
+ you need to configure enclave runtime as following:
+```json
+"annotations": {
+	"enclave.type": "intelSgx",
+	"enclave.runtime.path": "/usr/lib/liberpal-skeleton.so",
+	"enclave.runtime.args": "debug",
+	"enclave.attestation.ra_type": "EPID",
+	"enclave.is_product_enclave": "false",
+	"enclave.attestation.ra_epid_spid": "${EPID_SPID}",
+	"enclave.attestation.ra_epid_subscription_key": "${EPID_SUBSCRIPTION_KEY}",
+	"enclave.attestation.ra_epid_is_linkable": "false"
```

Comment:
Use bootstrap attestation configuration instead to hold your skeleton running, is it OK?